wagtail/contrib/settings/registry.py

```diff
@@ -10,21 +12,27 @@
 )
 from wagtail.admin.menu import MenuItem
 from wagtail.permission_policies import ModelPermissionPolicy
+from wagtail.utils.deprecation import RemovedInWagtail60Warning
 
 from .permissions import user_can_edit_setting_type
 
 
 class SettingMenuItem(MenuItem):
-    def __init__(self, model, icon="cog", classnames="", **kwargs):
-
+    def __init__(self, model, icon="cog", classname="", classnames="", **kwargs):
+        if classnames:
+            warn(
+                "The `classnames` kwarg for SettingMenuItem is deprecated - use `classname` instead.",
+                category=RemovedInWagtail60Warning,
+            )
+        classnames = classname or classnames
```

Comment:
perhaps?

```suggestion
        classname = classname or classnames
```

since we use use `classname` below

Reply:
Thank you.